packages/db/scripts/cmds/db-sql.js

```diff
@@ -64,29 +45,56 @@ export async function dbSqlCmd ({ reset, cargo, testing } = {}) {
   await client.query(configSql)
   await client.query(tablesSql)
 
-  if (cargo) {
-    if (testing) {
-      cargoSql = cargoSql.replace(
-        `
--- Create materialized view from cargo "aggregate_entries" table
-CREATE MATERIALIZED VIEW public.aggregate_entry
-AS
-SELECT *
-FROM cargo.aggregate_entries;`,
-        `
-CREATE MATERIALIZED VIEW public.aggregate_entry
-AS
-SELECT *
-FROM cargo.aggregate_entries 
-WHERE cid_v1 in ('bafybeiaj5yqocsg5cxsuhtvclnh4ulmrgsmnfbhbrfxrc3u2kkh35mts4e');
-`
-      )
-    }
-
+  // if testing or cargo fdw flag not set, you just get the schema, no fdw connection to dagcargo
+  if (testing || !cargo) {
+    await client.query(cargoSchema)
+  } else {
+    // Replace secrets in the FDW sql file
+    fdwSql = fdwSql.replace(":'DAG_CARGO_HOST'", `'${process.env.DAG_CARGO_HOST}'`)
+    fdwSql = fdwSql.replace(":'DAG_CARGO_DATABASE'", `'${process.env.DAG_CARGO_DATABASE}'`)
+    fdwSql = fdwSql.replaceAll(":'DAG_CARGO_USER'", `'${process.env.DAG_CARGO_USER}'`)
+    fdwSql = fdwSql.replaceAll(":'DAG_CARGO_PASSWORD'", `'${process.env.DAG_CARGO_PASSWORD}'`)
+    fdwSql = fdwSql.replaceAll(":'WEB3_STORAGE_USER'", `${process.env.WEB3_STORAGE_USER || 'CURRENT_USER'}`)
```

Comment:
```suggestion
    fdwSql = fdwSql.replaceAll(":WEB3_STORAGE_USER", `${process.env.WEB3_STORAGE_USER || 'CURRENT_USER'}`)
```

I don't know exactly how this works with normal SQL substitution, but I didn't include the quotes in the SQL because it shouldn't be a string...

Reply:
well, the others lines also use `'` and seem to be working. I added this because otherwise we had lint problems.

Reply:
so I guess we should change 

```
USER MAPPING FOR :WEB3_STORAGE_USER
```

to be 

```
USER MAPPING FOR ':WEB3_STORAGE_USER'
```

Reply:
The vars are like this because of psql variable interpolation:
https://www.postgresql.org/docs/9.2/app-psql.html#APP-PSQL-INTERPOLATION

In the SQL we have `:WEB3_STORAGE_USER` and it'll be replaced with `CURRENT_USER` (_not_ a string) (or whatever the env var is set to).

For the other variables, e.g. `:'DAG_CARGO_USER'` it is quoted because we want the substitution to be a string like: `'cargo'`.

I've changed the suggestion so no linting error.